app/pubspec.yaml

```diff
@@ -32,7 +32,7 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
-
+  
```

Comment:
Looks like we're gonna need the GitHub actions and pre-commit hooks now, it will precent these kinds of changes which are useless, etc. When we have the formatter, etc. up, we'll run it on all the code and clean it up.